core/src/test/java/nl/nn/adapterframework/stream/MessageTest.java

```diff
@@ -728,6 +740,17 @@ public void testNodeAsInputStream() throws Exception {
 		testAsInputStream(adapter);
 	}
 
+	@Test
+	public void testFileInputStreamAsInputStream() throws Exception {
+		URL url = this.getClass().getResource("/Message/testString.txt");
+		assertNotNull(url, "cannot find testfile");
+
+		File file = new File(url.toURI());
+		FileInputStream fis = new FileInputStream(file);
```

Comment:
Waarom gebruik je hier een FileInputStream? Gaat het om een repeatable inputstream of ?

Reply:
Dat was zodat ik kon zien hoe het loopt waarbij de markSupported false opleverd. Dat pad werd nog niet ingeslagen met de huidige coverage